ui/src/dashboards/components/Dashboard.js

```diff
@@ -23,7 +24,7 @@ const Dashboard = ({
   onSelectTemplate,
   showTemplateControlBar,
 }) => {
-  if (dashboard.id === 0) {
+  if (!dashboard) {
```

Comment:
Looks like this behavior change isn't tied to anything in this PR -- was this just old code?